src/eXpansion/Bundle/WidgetBestCheckpoints/Plugins/BestCheckpoints.php

```diff
@@ -96,8 +103,7 @@ public function onApplicationInit()
      */
     public function onApplicationReady()
```

Comment:
Can't we remove : ListenerInterfaceExpApplication ? I don't see it being used

Reply:
it was used before! I moved the stuff to setStatus... good catch though.